src/infobubble.js

```diff
@@ -1735,7 +1759,7 @@ InfoBubble.prototype.figureOutSize_ = function() {
   }
 
   this.contentContainer_.style['width'] = this.px(width);
-  this.contentContainer_.style['height'] = this.px(height);
+ // this.contentContainer_.style['height'] = this.px(height);
```

Comment:
I can't just ignore the line change unfortunately. &lt;yoda>Merge or do not merge, there is no ignore. &lt;/yoda>

If you remove this change I'll merge it in.